Misc/NEWS.d/next/Core and Builtins/2021-11-09-12-19-22.bpo-45764.8RLhWL.rst

```diff
@@ -0,0 +1,16 @@
+The parser now gives a better error message when leaving out the opening
+parenthesis ``(`` after a ``def``-statement::
+
+    >>> def f:
+      File "<stdin>", line 1
+        def f:
+             ^
+    SyntaxError: expected '('
+
+#.. section: Library #.. section: Documentation #.. section: Tests #..
```

Comment:
Could you remove the commented part of the news entry? Somehow it got I included in the file?

Reply:
done